Sources/PointFree/FeatureFlags.swift

```diff
@@ -0,0 +1,20 @@
+public struct Feature: Equatable {
```

Comment:
a type for feature flags. for now just wanna support things that admins have access to that others do not.